src/test/regress/expected/modification_correctness.out

```diff
@@ -0,0 +1,51 @@
+CREATE SCHEMA modification_correctness;
+CREATE TABLE test(a int, b int, c int unique, d int, e int);
+SELECT create_distributed_table('test', 'c');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+ALTER TABLE test DROP column b;
```

Comment:
maybe also drop 1 another column before `create distributed table` as well, we had weird edge cases on other code paths.